### ISSUE TITLE
Content updates for the API Programmer's Guide sample payload

### ISSUE THREAD

**Issue by jillian-hammer** [CDCgov/prime-reportstream#16336]
## Problem statement

To address user feedback, we updated some content on the API Programmer's Guide. See #16106 and #15997 for reference.

## What you need to know

- [Content doc](https://cdc.sharepoint.com/:w:/r/teams/ReportStream/_layouts/15/Doc.aspx?sourcedoc=%7B4563CA76-9BB8-40BD-B617-890BA194C4C0%7D&file=20241018_API_guide_content_updates.docx&action=default&mobileredirect=true) 

## Acceptance criteria

- [x] [Getting started](https://reportstream.cdc.gov/developer-resources/api/getting-started) page: Update step 1 text for accuracy, including `We can send you a file with fake data to review that will always successfully validate against the NIST HL7v2 validator.`
- [ ] [Sample payload](https://reportstream.cdc.gov/developer-resources/api/documentation/sample-payloads-and-output) page: Under the "Download sample file" button, write content that provides users a heads up about the 3 errors that they can expect with the sample file
- [ ] [Sample payload](https://reportstream.cdc.gov/developer-resources/api/documentation/sample-payloads-and-output) page: Update the "Response" code snippet, if needed

## To do

- [x] @victor-chaparro take a pass and review with @jillian-hammer 
- [x] @jillian-hammer  incorporate changes in the [content doc](https://cdc.sharepoint.com/:w:/r/teams/ReportStream/_layouts/15/Doc.aspx?sourcedoc=%7B4563CA76-9BB8-40BD-B617-890BA194C4C0%7D&file=20241018_API_guide_content_updates.docx&action=default&mobileredirect=true) accordingly
- [x] @jillian-hammer tag @chris-kuryak  for awareness to move FE implementation ticket from Blocked column (Implement updates to the API Programmer's Guide online content + PDF #16289)

**Comment by jillian-hammer** [comment]
update sent via Slack 10/29

> update! Victor sat down with me and we revised the API guide content on a call.
> TLDR:
> - We are removing the sample payloads page entirely, since it’s no longer applicable for the UP
> - Instead, we are including a callout to request an HL7 sample file within the body of the programmer’s guide with a link to the service request form.
> - I identified hyperlinks to the sample payloads page that we will have to remove in the same content doc. shouldn’t add too much of a lift for front-end, but worth calling out in case it bumpts it from 2 to 3pts.
> Action needed:
> - @victor-chaparro  can you approve this edit? “Currently, ReportStream can accept either HL7 or FHIR data. We can provide a sample HL7 file with fake data to review upon request.”
> - @chris-kuryak @MikaelahD13 upon your review and Victor’s approval, the FE eng ticket #16289 can be moved back to Sprint Backlog (rather than Blocked.)

**Comment by jillian-hammer** [comment]
completed - the front-end ticket is ready to be un-blocked and moved back to the sprint backlog @MikaelahD13 @chris-kuryak: Implement updates to the API Programmer's Guide online content + PDF #16289
